build/repos/opstools.repo

```diff
@@ -1,19 +1,18 @@
-# CentOS-OpsTools.repo
+# CentOS 9 OpenStack repos
 #
-# Please see http://wiki.centos.org/SpecialInterestGroup/OpsTools for more
-# information
+# Please see https://trunk.rdoproject.org/ and
+# https://trunk.rdoproject.org/centos9-caracal/report.html for more information
 
-[centos-opstools-testing]
-name=CentOS-OpsTools - testing repo
-baseurl=https://buildlogs.centos.org/centos/$releasever-stream/opstools/$basearch/collectd-5/
+[delorean-caracal-testing]
```

Comment:
Since the file is not about opstools only anymore, maybe it could be renamed? openstack-repos.repo or something like that?